Marlin/stepper.cpp

```diff
@@ -574,8 +574,11 @@ void set_stepper_direction() {
 // block begins.
 FORCE_INLINE void trapezoid_generator_reset() {
 
-  if (current_block->direction_bits != out_bits) {
+  static unsigned char last_extruder = 0;
```

Comment:
(a) Sorry but where would I be able to see that it should be that type? I looked in planner.h where the struct for the block is defined and there it is an unsigned char...
(b) Well yeah ok that would be super correct even if the first extruder is set correct when starting the interrupt.
[c] Sure I can rebase if we discussed all the stuff :)